### PR TITLE
First Sample App for Hybrid Support

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOQLBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
+﻿using System;
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOQLBuilder.cs]: Copyright?

File: SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOSLBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
+﻿using System;
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOSLBuilder.cs]: Copyright?

File: SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOSLReturningBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
+﻿using System;
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOSLReturningBuilder.cs]: Copyright?

File: SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/appbar.js [hunk 4]
```diff
@@ -0,0 +1 @@
+﻿
```
Comment (bhariharan) [inline SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/appbar.js]: What is this file?

File: SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/default.js [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+    }
+
+    function handlesyncupdate(sync) {
+        var manager = Salesforce.SDK.Hybrid.SmartSync;
+        var syncmanager = manager.SyncManager();
+
+        if (manager.Models.SyncStatusTypes.done != sync.Status) return;
+        switch (sync.SyncType) {
+            case manager.Models.SyncTypes.syncUp:
+                //RemoveDeleted();
```
Comment (bhariharan) [inline SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/default.js]: Why is this code commented out?

File: SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOQLBuilder.cs (hunk 1 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.SDK.Hybrid.SmartSync/Manager/SOQLBuilder.cs]: Will add it in all the files.

File: SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/appbar.js (hunk 4 above)
Comment (kchitalia) [inline SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/appbar.js]: Oversight. This file is not needed.

File: SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/default.js (hunk 5 above)
Comment (kchitalia) [inline SalesforceSDK/Sample.Salesforce.Hybrid.SmartSyncExplorer/js/default.js]: We don't need this, will remove the commented lines.

**Pull request merged**